config/settings.yml

```diff
@@ -227,6 +227,9 @@ default_projects_modules:
   - time_tracking
   - timelines
   - wiki
+  - meetings
+  - reporting_module
+  - backlogs
```

Comment:
You are referencing plugins here, @basiszwo. I guess this is in line with the approach to incorporate the plugins into the core. Even then, I think it would be better to keep that change within the plugins and ideally create an interface which can be used to make a module a default module. Otherwise, we will start to intermingle the different modules and responsibilities and the code will get worse.